.zuul.yaml

```diff
@@ -35,10 +35,46 @@
       - README*
       - .*/*.md
 
+- job:
+    name: functional-logging-tests-osp18
+    parent: telemetry-operator-multinode-autoscaling
+    description: |
+      Run the logging functional test on osp18+patched version
+    vars:
+      patch_openstackversions: true
```

Comment:
This is not needed for logging, since all the features we need are available in the openstack antelope containers in openstack-k8s-operators.
For autoscaling, there are some missing features on the antelope version, so we use different versions of heat and aodh which contain the features we need

Reply:
Done